docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md

```diff
@@ -54,7 +54,7 @@ Once the above steps are complete, you'll be able to log in (using your IDIR) to
 
 Our team has created the artifact below which captures the steps and teams involved in acquiring and mananaging GitHub Enterprise licenses. This version is intended for reference only, but we intend to create simplified, more focused versions for both end users and administrators.  
```

Comment:
recommend removing " but we intend to create simplified, more focused versions for both end users and administrators.  " and make a ticket for this work instead of putting it in docs... :)